lib/pages/bucket_page/bucket_page.dart

```diff
@@ -66,81 +95,47 @@ class BucketPage extends StatelessWidget {
           List<Object>? objects = state.items.objects;
           List<ListTile>? directoriesWidgets = [];
 
-          directories.forEach(
```

Comment:
a functional `forEach` is preferable if `for` is not explicitly required

Reply:
dart fix did that

Reply:
i reverted that decision and disabled that check